orcid-core/src/main/java/org/orcid/core/web/filters/AcceptFilter.java

```diff
@@ -53,10 +60,13 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             if (isValidAcceptType(contentType))
                 requestWrapper = new AcceptHeaderRequestWrapper(request, contentType);
             else
-                if (OrcidUrlManager.getPathWithoutContextPath(request).startsWith("/oauth/"))
+                if (OrcidUrlManager.getPathWithoutContextPath(request).startsWith("/oauth/")) {
```

Comment:
Hey @rcpeters,

For 1 path should never ever be null, but is a pretty small check we can add to prevent a null pointer exception that will not be logged (We could also add the logging line in case path is null)

For 2, the calls to the internal API are done like this: 

http://qa-5.orcid.org:8080/orcid-internal-api/member-info

So, OrcidUrlManager.getPathWithoutContextPath(request) will return just /member-info, what I want to check is that this is a request to the internal api, so, I need the request URI which is the full "/orcid-internal-api/member-info" and then verify that it starts with the internal api path.

Please let me know if am wrong. 

Thanks